Header header
string Class
float64 confidence
int64 xmin
int64 ymin
int64 xmax
int64 ymax